plugins/config-import.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-import
+spec:
+  version: "v0.6.0"
+  shortDescription: >-
+    Merge kubeconfigs from a file, stdin, or kubernetes secret.
+  homepage: https://github.com/rafi/kubectl-config-import
+  description: |
+    By default, an interactive fzf selection for namespace and secret is used
+    for user to select and merge as a kubeconfig. Using `-f` or `--file` you can
+    merge a file, or simply via stdin.
+
+    Examples:
+
+      kubectl config-import default remote-cluster-secret
+      kubectl config-import -f ~/Downloads/foo
+      cat foo | kubectl config-import
+      kubectl config-import --delete
+      kubectl config-import --edit
```

Comment:
FWIW typically these go out of date, consider listing them only in your README and -h/--help.